3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
```

Comment:
@rhcarvalho bash scripts are not intended for programming and we should not bend bash to look as programming language. The bash scripts should just run commands in order to automate administrator / developer tasks but not in programmatic way. Bash functions are nice but they should be used wisely (not as assemble() or main()).  

Bash is super hard to test (or unit test) and having monolithic functions just makes the problem worse. Ideally we should not be creating a 'SDK' or bash function libraries (we went that path in v2 and it was a mess).